packages/eslint-config-base/rules/style.js

```diff
@@ -250,7 +250,7 @@ module.exports = {
     // require multiline ternary
     // https://eslint.org/docs/rules/multiline-ternary
     // TODO: enable?
-    'multiline-ternary': ['off', 'never'],
+    'multiline-ternary': ['error', 'always'],
```

Comment:
```suggestion
    'multiline-ternary': ['error', 'always-multiline'],
```

This will only enforce it when the ternary is multiline.